app/views/admin/locations/_form.html.erb

```diff
@@ -1,107 +1,121 @@
-<%= form_for location, url: admin_location_path, method: 'put' do |f| %>
-  <% if location.errors.any? %>
+<% if location.errors.any? %>
 
-    <div class="row">
-      <div class="col-md-6">
-        <div class="error-summary t-errors ">
-          <h2 class="error-summary__heading">There's a problem</h2>
-          <ul class="error-summary__list">
-            <% error_messages_for(location, location.address).each do |attribute, attribute_errors| %>
-              <% attribute_errors.each do |attribute_error| %>
-                <li class="error-summary__list-item t-error-message">
-                  <a class="error-summary__link" href="#<%= attribute %>"><%= sanitize(attribute_error, :tags => []) %></a>
-                </li>
-              <% end %>
+  <div class="row">
+    <div class="col-md-6">
+      <div class="error-summary t-errors ">
+        <h2 class="error-summary__heading">There's a problem</h2>
+        <ul class="error-summary__list">
+          <% error_messages_for(location, location.address).each do |attribute, attribute_errors| %>
+            <% attribute_errors.each do |attribute_error| %>
+              <li class="error-summary__list-item t-error-message">
+                <a class="error-summary__link" href="#<%= attribute %>"><%= sanitize(attribute_error, :tags => []) %></a>
+              </li>
             <% end %>
-          </ul>
-        </div>
+          <% end %>
+        </ul>
       </div>
     </div>
-  <% end %>
-
-  <div class="form-group <%= 'form-group--error' if f.object.errors[:title].any? %>" id="title">
-    <%= f.label :title %>
-    <%= render 'error', error_messages: location.errors[:title] %>
-    <%= f.text_field :title, class: 'input-md-3 form-control t-location-title' %>
   </div>
+<% end %>
 
-  <%= f.fields_for(:address, location.address) do |a| %>
-    <div class="<%= 'form-group--error' if a.object.errors[:address_line_1].any? %>">
-      <div class="form-group" id="address_line_1">
-        <%= a.label :address_line_1, 'Address' %>
-        <%= render 'error', error_messages: a.object.errors[:address_line_1] %>
-        <%= a.text_field :address_line_1, class: 'input-md-3 form-control t-address-line-1' %>
-      </div>
-
-      <div class="form-group <%= 'form-group--error' if a.object.errors[:address_line_2].any? %>" id="address_line_2">
-        <%= render 'error', error_messages: location.address.errors[:address_line_2] %>
-        <%= a.text_field :address_line_2, class: 'input-md-3 form-control t-address-line-2' %>
-      </div>
+<% if policy(location).admin? %>
+  <div class="form-group <%= 'form-group--error' if f.object.errors[:organisation].any? %>" id="organisation">
+    <%= f.label :organisation, 'Twilio number' %>
+    <%= render 'error', error_messages: location.errors[:organisation] %>
+    <%= f.select :organisation, Location::ORGANISATIONS, {}, class: 'input-md-3 form-control t-organisation text-uppercase' %>
```

Comment:
I *think* this means that the default value will be the first organisation in `Location::ORGANISATIONS`. If so, could we `allow_blank` too to avoid this being missed and assigned to the wrong one?